packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/object-method-with-arrows/expected.js

```diff
@@ -16,9 +16,9 @@ class Class {
           var _this2 = this;
 
           this;
-          (function () {
-            _this2;
-          });
+          () => {
```

Comment:
I looks to me like the current async-to-generator transform is unnecessarily transforming arrows that are nested within nested function declarations and expressions.

Is there some reason why arrows nested within a nested function would need to be translated?